kalite/distributed/urls.py

```diff
@@ -38,6 +38,10 @@
     url(r'^securesync/', include(securesync.urls)),
 )
 
+
+# TODO: This should only be in DEBUG settings and the HTTP server should be
+# serving it otherwise. But the cherrypy service does not currently serve
+# media by itself, it uses the django application handlers below.
```

Comment:
Ah okay I didn't see that stuff in there... so I guess this one fires before the Django urlconf... but it's still a little peculiar not to have DEBUG=True... makes it look like we're intentionally using django's unrecommended static handler... Fixed the comment in 31bf9f2 -- but then I might actually just go ahead and add the DEBUG=True condition, right?